examples/deferred/main.rs

```diff
@@ -48,6 +47,11 @@ use std::time::{Instant};
 
 use noise::{Seed, perlin2};
 
+#[cfg(feature="metal")]
+pub use gfx::format::Depth32F as Depth;
```

Comment:
we could probably use it for all the backends, at least for the examples that require anything higher than GL 2.1